pandas/core/reshape/concat.py

```diff
@@ -344,6 +344,20 @@ def concat(
     Traceback (most recent call last):
         ...
     ValueError: Indexes have overlapping values: ['a']
+
+    Append a single row to the end of a ``DataFrame`` object.
+
+    >>> df7 = pd.DataFrame({'a': 1, 'b': 2}, index=[0])
+    >>> df7
+        a   b
+    0   1   2
+    >>> new_row = pd.Series([3])
+    >>> new_row
+    0   3
+    >>> pd.concat([df7, new_row.to_frame().T], ignore_index=True)
+         a    b    0
+    0  1.0  2.0  NaN
+    1  NaN  NaN  3.0
```

Comment:
ah apologies, hadn't realised it would look like this

how about
```
new_rows = pd.Series({'a': 3, 'b': 4})
```
so that then the concatenated version looks like
```
   a  b
0  1  2
0  3  4
```

Reply:
Changed as requested in 7d4a81fd3c10eeab9f38309bc508373329f0ce4b